src/components/CreateAreaObject.jsx

```diff
@@ -0,0 +1,42 @@
+import React, { useState } from "react";
+
+function CreateArea({ onAdd }) {
+  const [isExpanded, setExpanded] = useState(false);
+  const [note, setNote] = useState({ title: "", content: "" });
+
+  const handleChange = (e) => {
+    const { name, value } = e.target;
+    setNote((prevNote) => ({ ...prevNote, [name]: value }));
+  };
+
+  const handleSubmit = (e) => {
+    e.preventDefault();
+    onAdd(note);
```

Comment:
note[id] = uuid();

add the uuid package from npm 
generate an unique id that you will set here on the id property of the note
 just before adding it to state